gp-templates/project.php

```diff
@@ -178,8 +184,10 @@ function( $items ) {
 	<dt>
 		<?php gp_link_project( $sub_project, esc_html( $sub_project->name ) ); ?>
 		<?php gp_link_project_edit( $sub_project, null, array( 'class' => 'bubble' ) ); ?>
-		<?php if ( $sub_project->active ) {
-			echo "<span class='active bubble'>" . __( 'Active', 'glotpress' ) . "</span>"; } ?>
+		<?php
+		if ( $sub_project->active ) {
+			echo "<span class='active bubble'>" . __( 'Active', 'glotpress' ) . '</span>'; }
```

Comment:
```suggestion
			echo "<span class='active bubble'>" . __( 'Active', 'glotpress' ) . '</span>';
		}
```
The closing bracket should be on the next line.

Reply:
Of course :)
Also fixed the indent of the closing ?> tag
Mental note: so much escaping to do! :)